app/views/pages/get-started/installation-and-configuration.liquid

```diff
@@ -20,7 +20,7 @@ Make sure that you’ll verify your account by following the instructions you’
 
 ## Create an instance
 
-For your web app to work, you need to have a place where you will be running your code and storing files. On platformOS such place is called an _instance_.{% include 'alert/tip', content: 'Each separate site that you can create in the Partner Portal is called an **instance**.' %}
```

Comment:
Using simple language to be easier understandable be less tech savvy people.

Reply:
I think it's not too early to call things by their name. 
Calling an instance or an enviroment "a place" feels like oversimplification.

Reply:
Makes sense to find a better phrase than `a place`. In terms of `environment`, we actually use it also to distinguish staging from production - https://documentation.platformos.com/developer-guide/variables/context-variable#context-environment. How about going in the direction of "In order to begin working with platformOS, you need to create an Instance. ..." [ It will run your code? You will deploy your code to it? ... ]